src/simulation/dynamics/reactionWheels/_UnitTest/test_reactionWheelStateEffector_integrated.py

```diff
@@ -533,6 +545,6 @@ def test_setJs(show_plots):
 
 
 if __name__ == "__main__":
+    reactionWheelIntegratedTest(True,True,'BalancedWheels')
     # reactionWheelIntegratedTest(True,True,'BalancedWheels')
-    # reactionWheelIntegratedTest(True,True,'BalancedWheels')
-    test_setJs(False)
```

Comment:
Should we leave all unit test methods uncommented in our scripts or is it normal to keep just one at a time in main?

Reply:
It is normal to leave one at a time.  We have them there so we can quickly run a particular test function by uncommenting the right line.